tests/test_views.py

```diff
@@ -158,44 +165,194 @@ def test_view_poll_plugin_attached_to_page_should_return_related_page(self):
         self.assertEqual(response.context["querysets"][0].count(), 1)
         self.assertIn(page_content, response.context["querysets"][0])
 
-    def test_view_draft_and_published(self):
-        poll = PollFactory()
+    def test_extra_columns(self):
+        extra_column = ExtraColumn(lambda o: "{} test".format(o), "Test column")
+
+        with self.login_user_context(self.superuser), patch(
+            "djangocms_references.views.get_extra_columns", return_value=[extra_column]
+        ):
+            response = self.client.get(self.view_url)
+
+        self.assertEqual(response.context["extra_columns"], [extra_column])
+
+    def test_view_draft_filter_applied(self):
+        version1 = PageVersionFactory(
+            content__title="test1", content__language=self.language
+        )
+        version2 = PageVersionFactory(
+            content__title="test2", content__language=self.language, state=PUBLISHED
+        )
+
+        page_content1 = version1.content
+        page_content2 = version2.content
 
-        archived = PageVersionFactory()
         placeholder1 = PlaceholderFactory(
-            content_type=ContentType.objects.get_for_model(archived.content),
-            object_id=archived.content.id,
+            content_type=ContentType.objects.get_for_model(page_content1),
+            object_id=page_content1.id,
         )
-        add_plugin(placeholder1, "PollPlugin", "en", poll=poll, template=0)
 
-        version2 = archived.copy(archived.created_by)
-        version2.publish(archived.created_by)
-        version3 = version2.copy(version2.created_by)
+        poll1 = PollFactory()
+        # add poll plugin to page
+        add_plugin(placeholder1, "PollPlugin", "en", poll=poll1, template=0)
```

Comment:
There's some inconsistencies in the tests. This test will need an additional plugin as per the others.